### PR TITLE
fix: remove nightly references

### PR DESCRIPTION
**Pull request by petrSchreiber**

### CODE REVIEW

**Comment by petrSchreiber** [comment]
Hi,

this is an attempt to remove no-longer-necessary nightly references.

File: .travis.yml [hunk 1]
```diff
@@ -2,15 +2,15 @@ sudo: false
...
 addons:
   apt:
     packages:
       - aspell
       - aspell-en
 
 before_install:
-  - cargo install mdbook --vers '0.3.1' --debug --force
+  - cargo install mdbook --debug --force
   - cargo install mdbook-linkcheck --vers '0.3.1' --debug --force
```
Comment (orium) [inline .travis.yml]: Also remove `--vers` from here

Comment (petrSchreiber) [reply .travis.yml]: Just did, thank you for the tip 👍

File: examples/01_02_why_async/Cargo.toml [hunk 2]
```diff
@@ -7,4 +7,4 @@ edition = "2018"
 [lib]
 
 [dev-dependencies]
-futures-preview = { version = "=0.3.0-alpha.17", features = ["async-await", "nightly"] }
+futures = "=0.3.1"
```
Comment (cramertj) [inline examples/01_02_why_async/Cargo.toml]: These no longer need the leading `=`

**Comment by cramertj** [comment]
LGTM modulo the leading `=`

File: examples/01_02_why_async/Cargo.toml (hunk 2 above)
Comment (petrSchreiber) [reply examples/01_02_why_async/Cargo.toml]: Good point, addresssed - ready for review. Thank you

**Comment by cramertj** [comment]
Looks like this also has conflicts with master still

**Comment by petrSchreiber** [comment]
Changed to `futures = "0.3"`, resolved the merge conflict.

**Comment by petrSchreiber** [comment]
Hi @cramertj,

thank you for yesterdays review, it helped me to improve the commits.

May I ask you to consider having a look at this PR, before it gets out of sync with master, please?

Thank you,
_Petr_

**Comment by cramertj** [comment]
Looks great, thanks for the fixes!

**Pull request merged**